src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -331,4 +332,17 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		}
 		return children;
 	}
+
+	private getRenderedTextOutput(): string[] {
+		let textOutput: string[] = [];
+		let elements = this.getHtmlElements();
+		elements.forEach(element => {
+			if (element && element.innerText) {
+				textOutput.push(element.innerText);
```

Comment:
Any space concerns here for books with large amounts of output? Could considering making this lazy where it only fetches and stores the rendered output when requested.

Reply:
We're only getting the rendered text. It's a good point, but I'm not sure that we need to overindex on this. I'd also prefer for find results not to get hung up on some computation. If we see issues around memory usage, we can totally address in the future, but I think this is ok for now.